src/bridge-deployer/src/cli.rs

```diff
@@ -72,13 +80,18 @@ pub struct Cli {
 impl Cli {
     /// Runs the Bitfinity Deployer application.
     pub async fn run() -> anyhow::Result<()> {
+        let _ = dotenv::dotenv();
         let cli = Cli::parse();
 
         // Initialize tracing with the appropriate log level based on the verbosity setting.
         cli.init_tracing();
+        let identity = cli.init_identity()?;
+        info!(
+            "Using dfx identity with principal: {}",
+            identity.sender().unwrap()
```

Comment:
remove `unwrap`

Reply:
This function returns `String` error which cannot be cleanly converted to `anyhow::Error`. But this code should never fail with valid identity, so I just replaced `unwrap` with `expect`.